pages/linux/apt.md

```diff
@@ -4,28 +4,24 @@
 
 - Update list of packages and versions available. This should be run before running further apt commands:
 
-`apt update`
+`sudo apt update`
```

Comment:
`apt update` requires super-user privileges in order to function because it writes system-level files.

Regarding the presence of `sudo`, see bullet point 2 in https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines